### PR TITLE
Change to allow installation of latest RHSSO patch

### LINKED ISSUE #14: Support update to patch version of RH SSO
Currently the installation of RH-SSO via the keycloak role is only possible on baseline versions;
dedicated tasks are needed to perform cluster patch updates

### PR DESCRIPTION
**Pull request by guidograzioli**
- add tasks for running cli
- add tasks and config for downloading latest patch, and apply via cli

Fixes #14

### CODE REVIEW

File: roles/keycloak/tasks/rhsso_cli.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+- name: Verify server management interface is functional
+  ansible.builtin.uri:
+    url: "{{ keycloak_management_url }}"
+  register: result
+  until: result.status == 200
+  retries: 5
+  delay: 5
+
+- name: "Execute CLI query: {{ query }}"
```
Comment (rpelisse) [inline roles/keycloak/tasks/rhsso_cli.yml:17]: @guidograzioli as this collection depends on wilfly, will not reuse the jboss_cli.yml from it?

Comment (rpelisse) [reply roles/keycloak/tasks/rhsso_cli.yml:17]: I see now that you are using the --out-json. I think we actually need to enhance jboss_cli to use it too and returns directly json.

**Review by rpelisse — APPROVED** [review]

File: roles/keycloak/tasks/rhsso_cli.yml (hunk 1 above)
Comment (guidograzioli) [reply roles/keycloak/tasks/rhsso_cli.yml:17]: That was the initial idea of importing the eap role, but no matter what i tried, the jboss_cli distributed with keycloak/rhsso does not support to spit out json, so I am in the process of parsing DMR instead

**Review by rpelisse — APPROVED** [review]

**Comment by rpelisse** [comment]
@guidograzioli LGTM!

**Pull request merged**